modules/openapi-generator/src/main/resources/php-slim-server/composer.mustache

```diff
@@ -1,12 +1,13 @@
 {
   "minimum-stability": "RC",
   "require": {
-    "php": ">=5.5",
+    "php": ">=5.5.9",
     "slim/slim": "3.*",
     "tuupola/slim-basic-auth": "^3.0.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^4.8"
+    "phpunit/phpunit": "^4.8",
+    "overtrue/phplint": "^1.0"
```

Comment:
Can't find this line in `master` branch after merge. Looks like mege conflicts has been resolved incorrectly.

Reply:
My bad. I'll add it back shortly.

Reply:
Added back via 5ad471cc6. My apologies for the mistake when resolving the merge conflicts.